src/components/pages/PasswordReset/PasswordReset.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+import { useState } from 'react';
+
+import validateResetPassword from './resetPasswordValidation';
+import PasswordResetForm from './PasswordResetForm';
+import axiosWithHeader from '../../../utils/axiosWithHeaders';
+import { pathObj } from '../../../utils/generalVariables';
+import withRestrictedAccess from '../../hoc/withRestrictedAccess';
+
+const PasswordReset = ({ logOut }) => {
+  const [state, updateState] = useState({
+    newPassword: '',
+    confirmNewPassword: '',
+    errors: {},
+    resetting: false
+  });
+
+  const handleInputChange = (e, { name, value }) => {
+    updateState(prevState => ({ ...prevState, [name]: value }));
+  };
+
+  const handleSubmit = async () => {
+    updateState(prevState => {
+      return {
+        ...prevState,
+        resetting: true
+      };
+    });
+
+    const { newPassword, confirmNewPassword } = state;
+
+    const { errors, isValid } = await validateResetPassword({
+      newPassword,
+      confirmNewPassword
+    });
+
+    if (!isValid) {
+      toast.error('All fields are required');
+
+      return updateState(prevState => ({
+        ...prevState,
+        errors,
+        resetting: false
+      }));
+    }
+
+    axiosWithHeader()
```

Comment:
So I think to keep consistency this should be done in https://github.com/tieme-ndo/frontend/blob/master/src/utils/handlers/userHandlers.js

axiosWithHeader() is a good idea tho, but if that's incorporated it should also be used in all the handlers. We are now using this https://github.com/tieme-ndo/frontend/blob/master/src/utils/requestHeaders.js and passing the token in every of the handlers. Whatever path will be ok for me, just thinking about the consistency 👍